azure/azuredeploy.json

```diff
@@ -32,18 +37,88 @@
         "IsVersioningEnabled": true
       }
     },
+    {
+      "condition": "[not(equals(parameters('dbBackupStorageAccountName'), 'false'))]",
```

Comment:
It would be cleaner to have `"defaultValue": ""` and use `"condition": "[not(empty(parameters('dbBackupStorageAccountName')))]"` as the condition.

Reply:
Would need changing on lines 67 & 80 as well

Reply:
Doesn't work for storage accounts